xwiki-platform-core/xwiki-platform-wiki/xwiki-platform-wiki-ui/xwiki-platform-wiki-ui-mainwiki/src/main/resources/WikiManager/Translations.xml

```diff
@@ -226,15 +226,15 @@ platform.wiki.create.wizard.flavortemplate.label=Flavor / Template
 platform.wiki.create.wizard.flavortemplate.hint=Flavor or Template to use as initial content of the new wiki
 platform.wiki.create.wizard.flavortemplate.flavorTab=Flavor
 platform.wiki.create.wizard.flavortemplate.templateTab=Template
-platform.wiki.create.wizard.step.dw=After clicking on the "Create" button, the wiki will be created and the Distribution Wizard will be started to initialize your wiki. If you skip the wizard, your wiki will be empty.
+platform.wiki.create.wizard.step.dw=After clicking on the "Create" button, the wiki will be created and the Distribution Wizard would start to initialize your wiki. If you skip the wizard, your wiki will be empty.
```

Comment:
Not really consistent with the previous `wiki will be created` and the following `wiki will be empty`.

The previous version emphasized that the Distribution Wizard will be started so that the wiki is initialized. After DW starts, you will see it and you will have to make some decisions in order to initialize the wiki. DW does not initialize it immediately upon starting.

What about `will be started in order to help you initialize your wiki`?

Reply:
Done.